src/Gui/Language/FreeCAD.ts

```diff
@@ -4758,8 +4758,9 @@ Select a set and then press the button to view said configurations.</source>
     <message>
         <location filename="../DlgSettingsNavigation.ui" line="280"/>
         <source>Rotation orbit style.
-Trackball: moving the mouse horizontally will rotate the part around the y-axis
-Turntable: the part will be rotated around the z-axis.</source>
+Trackball: moving the mouse horizontally will rotate the part around the y-axis.
```

Comment:
Please revert the changes in the .ts file (it's autogenerated)

Reply:
Understood, it should be good now